hi-ml-cpath/environment.yml

```diff
@@ -84,11 +84,14 @@ dependencies:
     - azure-mgmt-keyvault==10.0.0
     - azure-mgmt-resource==21.1.0
     - azure-mgmt-storage==20.0.0
+    - azure-storage-blob==12.10.0
```

Comment:
Rather than modifying this directly, can you create a fresh locked environment by chaning `primary_deps.yml` and running the lock script? See comment at the start of the file.

Reply:
This was a workaround as I kept getting OutOfMemory errors when running the script but have since changed $TMP_DIR so should be able to try this again